documents/motions/08.2019.1 - Delegates.md

```diff
@@ -53,7 +51,7 @@ WCA Delegate is a role defined in the WCA Regulations to oversee official WCA Co
 8. Full Delegates
    1. Full Delegates are Staff of the WCA who have been appointed by the WCA to perform the role of WCA Delegate.
 9. Candidate Delegates
-   1. Candidate Delegates are External Staff of the WCA who have been appointed by the WCA to perform the role of WCA Delegate.
+   1. Candidate Delegates are Staff of the WCA who have been appointed by the WCA to perform the role of WCA Delegate.
```

Comment:
Is this line useful at all? We already have a definition in Motion 2.2019.1 I would either remove it or specify if they have Voting Rights or not. Something similar also happens in the Senior Delegates Motion.